packages/sing-box/build.sh

```diff
@@ -1,16 +1,18 @@
 TERMUX_PKG_HOMEPAGE=https://sing-box.sagernet.org
 TERMUX_PKG_DESCRIPTION="The universal proxy platform"
 TERMUX_PKG_LICENSE="GPL-3.0"
-TERMUX_PKG_MAINTAINER="kay9925@outlook.com"
-TERMUX_PKG_VERSION="1.2.3"
+TERMUX_PKG_MAINTAINER="@termux"
```

Comment:
Any reason why you stop naming yourself as maintainer?

Reply:
> Any reason why you stop naming yourself as maintainer?

Because Auto-update is enabled.

Reply:
Auto-updating package still needs maintainer(s). Please keep naming yourself.